contrib/kind-common.sh

```
@@ -1971,13 +1971,12 @@ install_ipsec() {
 # and makes sure the control-plane node is reachable by substituting 127.0.0.1
 # with the control-plane container's IP.
 run_script_in_container() {
-  local master_ip
   if [ "$PLATFORM_IPV4_SUPPORT" == true ]; then
-    master_ip=$("$OCI_BIN" inspect -f '{{ (index .NetworkSettings.Networks "kind").IPAddress }}' "${KIND_CLUSTER_NAME}-control-plane")
+    local master_ip=$("$OCI_BIN" inspect -f '{{range .NetworkSettings.Networks}}{{.IPAddress}}{{end}}' "${KIND_CLUSTER_NAME}-control-plane" | head -n 1)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)